src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapSeparatorAndInterfaces.java

```diff
@@ -1,6 +1,6 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+package com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap;
```

Comment:
>  InputOperatorWrap we use inner declared interfaces in InputSeparatorWrap

So we are only keeping this file because it has 2 interfaces used in another input? It also has 4 classes, which we don't even care about or use in any other tests?
This is another problem with our inputs.

If @romani is ok with this, we should move these 2 interfaces to `InputOperatorWrap` (as long as it doesn't hurt the test) and remove `InputSeparatorWrap` completely.